atom/common/api/object_life_monitor.cc

```diff
@@ -16,6 +16,7 @@ ObjectLifeMonitor::ObjectLifeMonitor(v8::Isolate* isolate,
       target_(isolate, target),
       weak_ptr_factory_(this) {
   target_.SetWeak(this, OnObjectGC, v8::WeakCallbackType::kParameter);
+  context_.SetWeak();
```

Comment:
Is this context used elsewhere? I'm not seeing other references to it, looks like it was added in https://github.com/electron/electron/commit/bd64f5ced2b5d2680d9bf2cea28ec038dfe62b49 but then was no longer used after https://github.com/electron/electron/commit/06cf0406fe8ddcef3a500a689952c77fdfb9c9a8.

@zcbenz do you think it could be completely removed now?